app/styles/app.scss

```diff
@@ -1,5 +1,6 @@
-@import "ember-bootstrap/bootstrap";
+@import 'ember-paper'
+// @import "ember-bootstrap/bootstrap";
 
-@import "par-statuses";
-@import "scorecards-table";
-@import "turns-form";
+// @import "par-statuses";
```

Comment:
I always prefer removal over commented out lines. You might remove these later. I'm going commit by commit.